### ISSUE TITLE
Where to install new versions of .Net Core SDK on Linux server?

### ISSUE THREAD

**Issue by ecoffman** [dotnet/docs#17072]
As I understand it, new versions of the .Net Core SDK can be installed over older versions of the .Net Core SDK. I think I read somewhere that newer versions can build projects that target previous versions. I've googled for this but can't seem to find an answer.

Is this correct? I ask because our Linux SA installs every update to the SDK in new locations making it complicated to manage builds within Jenkins.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a1200f60-5ad1-2b1e-067d-07f2dd442915
* Version Independent ID: 0e53d9c7-32ca-1341-f89a-a901a5636a9d
* Content: [.NET Core SDK and runtime dependencies - .NET Core](https://docs.microsoft.com/en-us/dotnet/core/install/dependencies?tabs=netcore31&pivots=os-linux#feedback)
* Content Source: [docs/core/install/dependencies.md](https://github.com/dotnet/docs/blob/master/docs/core/install/dependencies.md)
* Product: **dotnet-core**
* GitHub Login: @leecow
* Microsoft Alias: **leecow**

**Comment by Thraka** [comment]
TLDR:
Each SDK is standalone and you don't need older versions as long as you don't target a *specific* **major.minor.patch** version for your app. As such, you can safely delete an old SDK version and replace it with the latest patched version.

-------
When you target a specific version, you need that **major.minor** SDK version installed. The SDK a lot of pieces, including these two main pieces:

1. The `dotnet` CLI command for actions (like build, publish, add a package to a project, etc)
2. The framework libraries.

When you install a newer SDK, you want that as the `dotnet` CLI command, say the latest v3.1. You can still create apps for 2.1 with the 3.1 CLI as long as 2.1 SDK is installed. This is because the 2.1 app you're creating needs the framework libraries of 2.1 SDK to compile against.

You don't need every update to every minor SDK release though. You would just want the latest for the **major.minor** family. I would think that a build system would need the 3 supported SDKs, at whatever the latest patch is:

- 3.1
- 3.0
- 2.1

You only need one version of the SDK for each of those releases.
